backend/src/utils/chainRpcMap.ts

```diff
@@ -0,0 +1,6 @@
+export const chainRpcMap:any = {
```

Comment:
Create a file called constans or rename this file to constants, and try to move all hardcoded constants to the same file - so people can refer to constants for any addition or deletion.